ember-container-query/babel.config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [["@babel/preset-typescript"]],
+  "plugins": [
+    "@embroider/addon-dev/template-colocation-plugin",
+    ["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],
```

Comment:
I needed to add this line because I had used the `declare` keyword in the `{{container-query}}` modifier.

https://github.com/ijlee2/ember-container-query/blob/3.2.0/addon/modifiers/container-query.ts#L47